Швецов Артём

shvetsovart opened the pull request:
https://codesandbox.io/p/devbox/r6c8v9?layout=%257B%2522sidebarPanel%2522%253A%2522EXPLORER%2522%252C%2522rootPanelGroup%2522%253A%257B%2522direction%2522%253A%2522horizontal%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522id%2522%253A%2522ROOT_LAYOUT%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522cm0xuenae0007356kj2rtpzfp%2522%252C%2522sizes%2522%253A%255B70%252C30%255D%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522EDITOR%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522id%2522%253A%2522cm0xuenad0002356kv0copc7z%2522%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522SHELLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522id%2522%253A%2522cm0xuenad0004356kok4zx29g%2522%257D%255D%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522DEVTOOLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522id%2522%253A%2522cm0xuenad0006356kdr0npbep%2522%257D%255D%257D%255D%252C%2522sizes%2522%253A%255B50%252C50%255D%257D%252C%2522tabbedPanels%2522%253A%257B%2522cm0xuenad0002356kv0copc7z%2522%253A%257B%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522cm0xuenac0001356kxiemkl7u%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522FILE%2522%252C%2522filepath%2522%253A%2522%252Findex.html%2522%252C%2522state%2522%253A%2522IDLE%2522%252C%2522initialSelections%2522%253A%255B%257B%2522startLineNumber%2522%253A153%252C%2522startColumn%2522%253A8%252C%2522endLineNumber%2522%253A153%252C%2522endColumn%2522%253A8%257D%255D%257D%255D%252C%2522id%2522%253A%2522cm0xuenad0002356kv0copc7z%2522%252C%2522activeTabId%2522%253A%2522cm0xuenac0001356kxiemkl7u%2522%257D%252C%2522cm0xuenad0006356kdr0npbep%2522%253A%257B%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522cm0xuenad0005356k7z87zz6n%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522TASK_PORT%2522%252C%2522taskId%2522%253A%2522start%2522%252C%2522port%2522%253A5000%257D%255D%252C%2522id%2522%253A%2522cm0xuenad0006356kdr0npbep%2522%252C%2522activeTabId%2522%253A%2522cm0xuenad0005356k7z87zz6n%2522%257D%252C%2522cm0xuenad0004356kok4zx29g%2522%253A%257B%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522cm0xuenad0003356ksqtd9sfl%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522TASK_LOG%2522%252C%2522taskId%2522%253A%2522start%2522%257D%255D%252C%2522id%2522%253A%2522cm0xuenad0004356kok4zx29g%2522%252C%2522activeTabId%2522%253A%2522cm0xuenad0003356ksqtd9sfl%2522%257D%257D%252C%2522showDevtools%2522%253Atrue%252C%2522showShells%2522%253Atrue%252C%2522showSidebar%2522%253Atrue%252C%2522sidebarPanelSize%2522%253A15%257D

shvetsovart committed:
Create index.html

https://codesandbox.io/p/devbox/r6c8v9?layout=%257B%2522sidebarPanel%2522%253A%2522EXPLORER%2522%252C%2522rootPanelGroup%2522%253A%257B%2522direction%2522%253A%2522horizontal%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522id%2522%253A%2522ROOT_LAYOUT%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522cm0xuenae0007356kj2rtpzfp%2522%252C%2522sizes%2522%253A%255B70%252C30%255D%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522EDITOR%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522id%2522%253A%2522cm0xuenad0002356kv0copc7z%2522%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522SHELLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522id%2522%253A%2522cm0xuenad0004356kok4zx29g%2522%257D%255D%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522DEVTOOLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522id%2522%253A%2522cm0xuenad0006356kdr0npbep%2522%257D%255D%257D%255D%252C%2522sizes%2522%253A%255B50%252C50%255D%257D%252C%2522tabbedPanels%2522%253A%257B%2522cm0xuenad0002356kv0copc7z%2522%253A%257B%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522cm0xuenac0001356kxiemkl7u%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522FILE%2522%252C%2522filepath%2522%253A%2522%252Findex.html%2522%252C%2522state%2522%253A%2522IDLE%2522%252C%2522initialSelections%2522%253A%255B%257B%2522startLineNumber%2522%253A153%252C%2522startColumn%2522%253A8%252C%2522endLineNumber%2522%253A153%252C%2522endColumn%2522%253A8%257D%255D%257D%255D%252C%2522id%2522%253A%2522cm0xuenad0002356kv0copc7z%2522%252C%2522activeTabId%2522%253A%2522cm0xuenac0001356kxiemkl7u%2522%257D%252C%2522cm0xuenad0006356kdr0npbep%2522%253A%257B%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522cm0xuenad0005356k7z87zz6n%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522TASK_PORT%2522%252C%2522taskId%2522%253A%2522start%2522%252C%2522port%2522%253A5000%257D%255D%252C%2522id%2522%253A%2522cm0xuenad0006356kdr0npbep%2522%252C%2522activeTabId%2522%253A%2522cm0xuenad0005356k7z87zz6n%2522%257D%252C%2522cm0xuenad0004356kok4zx29g%2522%253A%257B%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522cm0xuenad0003356ksqtd9sfl%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522TASK_LOG%2522%252C%2522taskId%2522%253A%2522start%2522%257D%255D%252C%2522id%2522%253A%2522cm0xuenad0004356kok4zx29g%2522%252C%2522activeTabId%2522%253A%2522cm0xuenad0003356ksqtd9sfl%2522%257D%257D%252C%2522showDevtools%2522%253Atrue%252C%2522showShells%2522%253Atrue%252C%2522showSidebar%2522%253Atrue%252C%2522sidebarPanelSize%2522%253A15%257D

mokhov commented on index.html:
```diff
@@ -0,0 +1,153 @@
+        </article>
+    
+        <footer>
+            <h5>HOGWARTS SCHOOL of HTML & CSS</h5>
```
Мне кажется что футер не лучшее место для загловков

mokhov commented on index.html:
```diff
@@ -0,0 +1,153 @@
+                </p>
+                
+                <p>Yours sincerely,</p>
+                <img src="signature.png" alt="signature" width="150px">
```
Можно не указывать "px"

mokhov commented on index.html:
```diff
@@ -0,0 +1,153 @@
+                        <li>Last year I ran a marathon</li>
+                        <li>The A in History I got for:
+                            <ol>
+                                <li>building a website <ruby><span style="color:grey">(on Tilda)</span><rt>[I'm sorry]</rt></ruby></li>
```
👍

mokhov commented on index.html:
```diff
@@ -0,0 +1,153 @@
+                J.S.(sorc.), International Confed. of Frontenders)</h6>
+            <hr>
+            <address>
+                <p><mark>Please contact me via email or Telegram for further communication</mark> <small>(my roommates get scared when something flies through our window):</small></p>
```
Тег small не запрещен, но считается устаревшим

mokhov reviewed (commented):
Есть что исправить

shvetsovart committed:
Update index.html

Fixed some mistakes, added something new

mokhov commented on index.html:
```diff
@@ -0,0 +1,162 @@
+                        <li>Last year I ran a marathon</li>
+                        <li>The A in History I got for:
+                            <dl>
+                                <dt>- Building a website <ruby><span style="color:grey">(on Tilda)</span><rt>[I'm sorry]</rt></ruby></dt>
```
Не самое лучшее применение dd-dt. Обычно их применяют в списке терминов.

mokhov commented on index.html:
```diff
@@ -0,0 +1,162 @@
+                </p>
+                
+                <p>Yours sincerely,</p>
+                <img src="signature.png" alt="signature" width="150">
```
alt текст может быть понятнее, например «Подпись автора»

mokhov commented on index.html:
```diff
@@ -0,0 +1,162 @@
+
+            <section> <!-- Main part -->
+                <p>
+                    I am not the best <abbr>CT ITMO</abbr> (Computer Technologies faculty in ITMO [ITMO is not <br>
```
У abbr стоит указывать атрибут title https://developer.mozilla.org/en-US/docs/Web/HTML/Element/abbr

mokhov reviewed (commented):
Ещё чуть-чуть

shvetsovart committed:
Update index.html

Fixed: добавил title в abbr, сделал alt-текст у подписи более понятным, исправил dd-dt и вообще весь список на ol-ul.
Добавил blackquote для цитаты.
Сделал букву "Я" красной.

mokhov commented:
Есть что ещё улучшить. Но ОК, принято

shvetsovart commented:
Спасибо! А можете, пожалуйста, вкратце сказать, что ещё можно улучшить?

чт, 12 сент. 2024 г. в 18:14, Oleg Mokhov ***@***.***>:

> Есть что ещё улучшить. Но ОК, принято
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/itmo-2024/task1/pull/1#issuecomment-2346580319>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AGIY53IRIE6G5E4S3VOKQSLZWGVWTAVCNFSM6AAAAABOA4UZ26VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDGNBWGU4DAMZRHE>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
